_posts/2020-12-09-november-rubygems-update.md

```diff
@@ -0,0 +1,54 @@
+* merged a PR to support the new signin endpoints. [#3840](https://github.com/rubygems/rubygems/pull/3840)
+* merged a PR to improve the `-C` flag to gem build. [#3983](https://github.com/rubygems/rubygems/pull/3983)
+* added a fix to slightly improve some "**gem not found**" error messages. [#4019](https://github.com/rubygems/rubygems/pull/4019)
+* fixed an intermittent spec failure. [#4060](https://github.com/rubygems/rubygems/pull/4060)
+* fixed an issue with changelog generation. [#4059](https://github.com/rubygems/rubygems/pull/4059)
+* fixed an issue with nested bundler invocations. [#4063](https://github.com/rubygems/rubygems/pull/4062)
+* fixed a discrepancy between executing with or without bundle exec. [#4063](https://github.com/rubygems/rubygems/pull/4063)
+* added more descriptive errors about default network errors. [#4061](https://github.com/rubygems/rubygems/pull/4061)
+* fixed a CI issue that appeared under Windows. [#4068](https://github.com/rubygems/rubygems/pull/4068)
+* merged a tweak to the bundle gem default skeleton. [#4066](https://github.com/rubygems/rubygems/pull/4066)
+* Extend `gem` DSL with a `force_ruby_platform` option. [#4049](https://github.com/rubygems/rubygems/pull/4049)
+
+In November, [RubyGems](https://github.com/rubygems/rubygems) gained [58 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2020-11-01%7D...master@%7B2020-11-30%7D), contributed by 10 authors. There were 816 additions and 426 deletions across 79 files.
+
+
+## RubyGems.org News
+This month, we coordinated with Fastly support to enable Globalsign certs and AAAA records in our TLS config. We updated [RubyGems CLI gem signin](https://github.com/rubygems/rubygems/pull/3840) according to changes requested in a review and also made the following fixes and improvements:
+
+* added a new way to match RubyGems versions using the `build-arg` in docker image. [#2548](https://github.com/rubygems/rubygems.org/pull/2548)
+* fixed failing tests in `shoulda-matchers` update and reported an issue of invalid objects should `belong_to` tests on `shoulda-matchers` repo. [#1375]( https://github.com/thoughtbot/shoulda-matchers/issues/1375)
+* investigated `DelegationError` for ownership records with nil `user_id`.
+* resolved HO#200743 email spooling by setting DMARC policy to quarantine.
+* resolved HO#795112 and disallowed duplicate canonical version numbers. [#2559](https://github.com/rubygems/rubygems.org/pull/2559)
+* read privacy policy of other package manager websites and researched the requirements for CCPA and GDPR.
+* add `Pagerduty` integration for Cloudwatch ALB alerts.
+
+As always, we continue to fix bugs, review and merge PR's and reply to support tickets.
+
+In November, [RubyGems.org](https://github.com/rubygems/rubygems.org) gained [11 new commits](https://github.com/rubygems/rubygems.org/compare/master@%7B2020-11-01%7D...master@%7B2020-11-30%7D), contributed by  authors. There were 63 additions and 4 deletions across 8 files.
```

Comment:
I am seeing the following, is it because of timezones?
![Screenshot-20201210103602-1248x144](https://user-images.githubusercontent.com/7680662/101724643-16ce5c80-3ad5-11eb-9539-8745941f44ea.png)

Reply:
Thank you! Yeah, I see I will update the stats. When I ran the script it was the last week so some commits was missing.